.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,42 @@
+name: Build and Publish to NPM
+on:
+    push:
+        branches: [master, development]
+
+permissions:
+    contents: read # for checkout
+
+jobs:
+    build_and_publish_npm:
+        name: Release
+        runs-on: ubuntu-latest
+        environment: Production
```

Comment:
```suggestion
```

Do we need this? we don't have different environment it's just production 🤔

Reply:
Its better to keep it here for vite, semantic release and other libraries to know this is for a production build

Reply:
Ok then let's keep it